ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.8.yaml

```diff
@@ -12,7 +12,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-8-release-openshift-4.8
+    tag: rhel-8-release-openshift-4.3
```

Comment:
I dont understand, why we need 4.3 for ocp 4.8 ?

Reply:
neither do I, but the 4.8 tag does not exist